qml/model/CallHistory.qml

```diff
@@ -139,38 +138,52 @@ Db8Model {
 
         if( endedVoiceCall.isIncoming ) {
             newCallItem.from = personDetails;
-            newCallItem.to.push ({ addr: "", service: "com.palm.telephony" });
+            newCallItem.to.push({ addr: "", service: "com.palm.telephony" });
             newCallItem.type = endedVoiceCall.duration > 0 ? "incoming" : "missed";
```

Comment:
There is a 3rd scenario which is called "rejected" for which the duration is also 0. Did you think about that?